src/models/bonw15/internal-overview.md

```diff
@@ -0,0 +1,27 @@
+# Bonobo WS (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- LCD panel connector is highlighted in light green
+- Webcam/microphone connector is highlighted in cyan
+- Keyboard adhesive access point is highlighted in pink
+- Wireless card (under the innermost SSD) is highlighted in navy
+- M.2 SSDs are highlighted in dark green
+    - Both slots support PCIe NVMe Gen 4
+- Fan connectors are highlighted in orange
+- CMOS battery connector is highlighted in purple
+- Speaker connectors are highlighted in yellow
+    - Right speaker connector (on the left) is under SSD slot 3
```

Comment:
>Right speaker connector (on the left)
Should this be "Left speaker connector (on the right)" since it's the left channel speaker, but it's on the right when the machine is on its back like the photo?